kairon/shared/actions/utils.py

```
@@ -1132,4 +1134,24 @@ def trigger_rephrase(bot: Text, llm_type: str, text_response: Text):
+            first_name = "Team kAIron"
+            asyncio.create_task(
+                MailUtility.format_and_send_mail(mail_type=mail_type,
+                                                   email=email,
+                                                   first_name=first_name,
+                                                   stack_trace=stack_trace,
+                                                   slot_values=slot_values,
+                                                   bot_name=bot_name,
+                                                   action_name=action_name,
+                                                   user_query_history=user_query_history
+                                                   )
+            )
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)